controllers/ibu_controller_test.go

```diff
@@ -644,7 +753,7 @@ func TestValidateStageTransisions(t *testing.T) {
 					utils.ConditionTypes.UpgradeCompleted,
 					utils.ConditionReasons.Failed,
 					metav1.ConditionFalse,
-					"Rollback requested",
+					"Upgrade failed",
```

Comment:
@donpenney I think you specifically asked for "rollback requested"?

Reply:
If it's a user-driven rollback, it's not a "failed upgrade" - it's an aborted upgrade. So I don't think we would want to set the message to "failed", as it could be misleading.